### PR TITLE
add Syndaver AXI machine

### PR DESCRIPTION
**Pull request by synsuiter**
Add Syndaver AXI Machine and Extruder Profile

### CODE REVIEW

File: resources/definitions/syndaveraxi.def.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+         },
+        "machine_end_gcode": {
+            "default_value": "M400                                               ; wait for moves to finish\nM140 S{material_part_removal_temperature}          ; start bed cooling\nM104 S0                                            ; disable hotend\nM107                                               ; disable fans\nM117 Cooling please wait\nG91                                                ; relative positioning\nG1 Z5 F3000                                        ; move Z up 5mm so it wont drag on the print\nG90                                                ; absolute positioning\nG1 X5 Y5 F3000                                     ; move to cooling position\nM190 R{material_part_removal_temperature}          ; wait for bed to cool down to removal temp\nG1 X145 Y260 F1000                                 ; present finished print\nM140 S{material_keep_part_removal_temperature_t}   ; keep temperature or cool down\nM77                                                ; End LCD Print Timer\nM18 X Y E                                          ; turn off x y and e axis\nM117 Print Complete."
```
Comment (konskarm) [inline resources/definitions/syndaveraxi.def.json:43]: The settings "material_part_removal_temperature" and "material_keep_part_removal_temperature_t" mentioned in the end gcode don't exist. 
Cura complains: 
```
Unable to replace 'material_part_removal_temperature' placeholder in start/end g-code
Unable to replace 'material_keep_part_removal_temperature_t' placeholder in start/end g-code
```

**Comment by synsuiter** [comment]
Thank you @konskarm, those are now defined to the desired values.

File: resources/definitions/syndaveraxi.def.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        }
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "AXI" },
+        "machine_shape": { "default_value": "rectangular"},
+        "machine_heated_bed": { "default_value": true },
+        "machine_width": { "default_value": 280 },
+        "machine_depth": { "default_value": 280 },
+        "machine_height": { "default_value": 286 },
```
Comment (konskarm) [inline resources/definitions/syndaveraxi.def.json:26]: A bit of a nitpicking, but the official product dimensions are 280x280x285. Is the extra milimeter in the height intended?

**Comment by synsuiter** [comment]
@konskarm, I talked with our engineers, no reason for this Z difference. It has been corrected.

**Comment by konskarm** [comment]
I cannot spot any other issues with it. The printer slices without a problem, so I am merging this.

**Comment by synsuiter** [comment]
@konskarm Thank you, have a good one.

**Pull request merged**